CHANGELOG.md

```diff
@@ -7,20 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+
+### Added
+
+### Fixed
+
+## [0.7.0] - 2024-03-12
```

Comment:
Can add a markdown link for 0.7.0 and update the Unreleased markdown link -- https://github.com/partiql/partiql-lang-rust/blob/release-v0.7.0/CHANGELOG.md?plain=1#L221